### PR TITLE
KGraphViewer

### PR DESCRIPTION
**Pull request by Mailaender**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://invent.kde.org/teams/flathub/issues/-/issues/14
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)* https://invent.kde.org/graphics/kgraphviewer/-/merge_requests/8 https://invent.kde.org/graphics/kgraphviewer/-/commit/c007946a889d6b27d4c0d68e9fb7dde99d95a368.patch

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by hfiguiere** [comment]
cc @flathub/kde

File: org.kde.kgraphviewer.json [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "command": "kgraphviewer",
+    "rename-icon": "kgraphviewer",
+    "finish-args": [
+        "--device=dri",
+        "--share=ipc",
+        "--socket=cups",
+        "--socket=wayland",
+        "--socket=x11"
```
Comment (hfiguiere) [inline org.kde.kgraphviewer.json:13]: either X11 or wayland + fallback

File: org.kde.kgraphviewer.json [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+                        "project-id": 1249,
+                        "stable-only": true,
+                        "url-template": "https://gitlab.com/api/v4/projects/4207231/packages/generic/graphviz-releases/$version/graphviz-$version.tar.xz"
+                    }
+                }
+            ]
+        },
+        {
+            "name": "kgraphviewer",
+            "buildsystem": "cmake",
```
Comment (hfiguiere) [inline org.kde.kgraphviewer.json:60]:
```suggestion
            "buildsystem": "cmake-ninja",
```

File: org.kde.kgraphviewer.json [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.kde.org/stable/kgraphviewer/2.4.3/kgraphviewer-2.4.3a.tar.xz",
+                    "sha256": "54e0c218453912ea82ad9a96edfb2d4dfd8cb15f6a567ef843b4ef97d07bd7c0"
+                },
+                {
+                    "type": "file",
+                    "path": "logo.png"
```
Comment (hfiguiere) [inline org.kde.kgraphviewer.json:72]: was this issue submitted upstream?

File: org.kde.kgraphviewer.json [hunk 4]
```diff
@@ -0,0 +1,77 @@
+{
+    "id": "org.kde.kgraphviewer",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
```
Comment (hfiguiere) [inline org.kde.kgraphviewer.json:4]: 22.08 maybe?

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: org.kde.kgraphviewer.json (hunk 3 above)
Comment (Mailaender) [reply org.kde.kgraphviewer.json:72]: Yes, see above.

File: org.kde.kgraphviewer.json (hunk 4 above)
Comment (Mailaender) [reply org.kde.kgraphviewer.json:4]: I was told it is not ready yet.

**Comment by Mailaender** [comment]
bot, build org.kde.kgraphviewer

**Comment by Mailaender** [comment]
bot, build org.kde.kgraphviewer

**Pull request closed without merging**